spec/unit/puppet/parser/functions/enclose_ipv6_spec.rb

```diff
@@ -49,8 +53,8 @@
   end
 
   it "should embrace ipv6 adresses within an array of ip addresses" do
-    result = scope.function_enclose_ipv6([['127.0.0.1','fe80::1','[fe80::1]']])
-    expect(result).to(eq(['127.0.0.1','[fe80::1]','[fe80::1]']))
+    result = scope.function_enclose_ipv6([['127.0.0.1','fe80::1','[fe80::2]']])
+    expect(result).to(eq(['127.0.0.1','[fe80::1]','[fe80::2]']))
   end
```

Comment:
It would be appreciated if these changes could be explained in the context of wildcard vhosts. Thanks!